wheels/build/wheels.yml

```diff
@@ -1,6 +1,10 @@
 ---
 
 packages:
+    biopython:
+        version: 1.66
+        prebuild:
+            wheel: "git clone -b linux-wheels https://github.com/natefoo/pip && cd pip && for py in /python/cp*-`uname -m`; do ${py}/bin/python setup.py install && ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```

Comment:
A second correction: this works for manylinux1 but /python/cp* under the OS X image does not include pip, so this should do it:

```yaml
wheel: "for py in /python/cp*-`uname -m`; do ${py}/bin/easy_install pip; ${py}/bin/pip install --upgrade pip ; ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```

We should probably just install pip in the build pythons in the image, so I created #108 for this.